hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -675,6 +675,13 @@ class Emit[C](
         presentC(const(true))
       case False() =>
         presentC(const(false))
+      case Consume(value) => {
+        val iec = emitI(value)
+        iec.map(cb){pc =>
```

Comment:
There's no need to bind `iec` to a variable here. `emitI(value).map` would be sufficient.